index.rst

```diff
@@ -97,6 +97,7 @@ Contents
    testnet/rewards.rst
    testnet/faq.rst
    testnet/community-resources.rst
+   testnet/all-config.rst
```

Comment:
I'd rather keep FAQ and community resources at the end.

Reply:
done